addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -316,79 +316,57 @@
                     <u><a class="h4 text-primary" t-out="event.name" t-att-href="event.website_url" /></u>
                 </div>
             </div>
-            <div t-if="attendees" class="row mb-3">
-                <div class="col-12 mb-2">
-                    <a class="btn btn-primary" title="Download All Tickets" target="_blank"
-                        t-attf-href="/event/{{ event.id }}/my_tickets?registration_ids={{ attendees.ids }}&amp;tickets_hash={{ event._get_tickets_access_hash(attendees.ids) }}">
-                        Download Tickets <i class="ms-1 fa fa-download"/>
-                    </a>
-                </div>
-            </div>
-            <div class="row mb-3 o_wereg_confirmed_attendees">
-                <div class="col-md-4 col-xs-12 mt-3" t-foreach="attendees" t-as="attendee">
-                    <div class="d-flex flex-column">
-                        <span t-if="attendee.name"  class="fw-bold text-truncate">
-                            <t t-out="attendee.name"/>
-                        </span>
-                        <span t-if="attendee.email" class="text-truncate">
-                            <i class="fa fa-envelope me-2"/>
-                            <t t-out="attendee.email"/>
-                        </span>
-                        <span t-if="attendee.phone">
-                            <i class="fa fa-phone me-2"/>
-                            <t t-out="attendee.phone"/>
-                        </span>
-                        <span>
-                            <i class="fa fa-ticket me-2"/>
-                            <t t-if="attendee.event_ticket_id">
-                                <t t-out="attendee.event_ticket_id.name"/> (Ref: <t t-out="attendee.id"/>)
-                            </t>
-                            <t t-else="">Ref: <t t-out="attendee.id"/></t>
-                        </span>
-                    </div>
-                </div>
-            </div>
-            <div class="row mb-3">
-                <div class="col">
-                    <div class="row">
-                        <div class="col-2 col-md-1">
-                            <b>Start</b>
-                        </div>
-                        <div class="col ps-0">
-                            <span t-out="event.date_begin_located"/>
-                            (<span t-out="event.date_tz"/>)
+            <div class="row">
+                <div class="d-flex flex-column justify-content-between flex-lg-row">
```

Comment:
It is strange to mix both grid and flex display ... especially you seem to use a `row` without `col` inside, so do we need the row ?